tests/Unit/ControlSystem/Test_Averager.cpp

```diff
@@ -48,8 +51,8 @@ SPECTRE_TEST_CASE("Unit.ControlSystem.Averager.Linear",
       // The exponential averager uses finite differencing to approximate the
       // derivatives. The second derivative is only a first order approximation,
       // which is why we enforce a less stringent check here.
-      CHECK(fabs(result_t[2][0] - analytic_func[2]) <= 1.0e-12);
-      CHECK(fabs(result_f[2][0] - analytic_func[2]) <= 1.0e-12);
+      CHECK(custom_approx(fabs(result_t[2][0])) == analytic_func[2]);
```

Comment:
Is the `fabs` here intentional?  This is testing a different thing than the previous version.

Reply:
That doesn't need to be there, thanks for pointing that out. That was an old fix for a failing `CHECK` -- the value should be zero, so was sometimes small and negative (hence `fabs`) -- but the appropriate fix was using `custom_approx` (as you suggested) with a `scale(1.0)` (otherwise it will fail, since I think `CHECK` does something like epsilon*(scale + min{`result_t[2][0]`,`analytic_func[2]`}) and scale defaults to 0)